src/Compilation.zig

```diff
@@ -414,24 +440,25 @@ pub const AllErrors = struct {
                         }
                     }
                     for (src.notes) |note| {
-                        try note.renderToStdErrInner(ttyconf, stderr_file, "note:", .Cyan, indent);
+                        try note.renderToStdErrInner(ttyconf, stderr_file, "note", .Cyan, indent);
                     }
                 },
                 .plain => |plain| {
```

Comment:
I wasn't sure what this was for so I didn't touch it (apart from the refactor) but I'm assuming output printed with this variant only uses one line anyway, right? But I don't know, does anyone know what this variant is for and whether I should use `src.writeMsg` here too?

Reply:
`.plain` is only used in a few cases where there is no source line, you can safely assume them to never include new lines.